ClosedXML.Report/Excel/Subtotal.cs

```diff
@@ -237,7 +237,7 @@ private void MoveSummary(MoveData moveData)
                 return;
 
             trgtRng.Clear();
-            fcell.Value = srcRng;
+            fcell.CopyFrom(srcRng); //fcell.Value = srcRng;
```

Comment:
Please remove commented text. We have git )

Reply:
I removed my comments and pushed it again.

Reply:
I cloned the branch and got the next error: 

```
 ClosedXML.Report.Tests.csproj: [NU1605] Warning As Error: Detected package downgrade: System.Linq.Dynamic.Core from 1.3.1 to 1.2.20. Reference the package directly from the project to select a different version. 
 ClosedXML.Report.Tests -> ClosedXML.Report -> System.Linq.Dynamic.Core (>= 1.3.1) 
 ClosedXML.Report.Tests -> System.Linq.Dynamic.Core (>= 1.2.20)
```

Reply:
Fixed error and run all tests:

![image](https://user-images.githubusercontent.com/3437698/229068534-fef9b6a1-a781-4cca-bee5-5b2d5de9a297.png)

Reply:
Cannot say for all tests, but for this the test should be fixed:
![image](https://user-images.githubusercontent.com/7596548/230316936-8bfca64f-660c-45c7-a2f4-d9669b55fe8d.png)

It has to do with Cell.Value now being an XLCellValue type where before was object.